module8/module8.js

```diff
@@ -0,0 +1,27 @@
+let fs = require("fs");
+const path = "./file.txt";
+function checkFile() {

```

Comment:
Ok, maybe I did not specify it in the document. I would like you the change that function in a way that it will stop if the file exist and keep checking every 5s if there file is not there. Lets say max 20s. Right now this function is running forever no matter if the file is or isn't there.